packages/remix-dev/__tests__/readConfig-test.ts

```diff
@@ -27,6 +27,7 @@ describe("readConfig", () => {
         "cacheDirectory": Any<String>,
         "devServerBroadcastDelay": 0,
         "devServerPort": 8002,
+        "devServerShouldShowLoadingSpinner": true,
```

Comment:
Let's be careful about adding root keys to the config. Can we try to come up with a key name for development overlay configuration? We will probably need a few more keys in there in the future for surfacing uncaught errors and whatnot.

Reply:
not sure if I got this correctly, do you mean something like `devServerOptions` for example that combines `boadcastDelay`,  `port`, `shouldShowLoadingSpinner` options?

Reply:
I wonder if we can take some naming inspiration from things like Vite also.

I would do something like this:

```
devServer: {
  rebuildingIndicator: true
}
```